SpatialGDK/Source/SpatialGDKServices/Private/SpatialGDKServicesModule.cpp

```diff
@@ -137,16 +140,34 @@ void FSpatialGDKServicesModule::ExecuteAndReadOutput(const FString& Executable,
 	FPlatformProcess::ClosePipe(0, WritePipe);
 }
 
+void FSpatialGDKServicesModule::SetProjectName(const FString& InProjectName)
+{
+	FString SpatialFileName = TEXT(SpatialOSConfigFile);
+	FString SpatialFileResult;
+
+	JsonParsedSpatialFile->SetStringField("name", InProjectName);
+	
+	TSharedRef<TJsonWriter<>> JsonWriter = TJsonWriterFactory<>::Create(&SpatialFileResult);
+	if (!FJsonSerializer::Serialize(JsonParsedSpatialFile.ToSharedRef(), JsonWriter))
+	{
+		UE_LOG(LogSpatialGDKServices, Error, TEXT("Write project name failed! Can not Serialize content to json file!"));
```

Comment:
```suggestion
		UE_LOG(LogSpatialGDKServices, Error, TEXT("Failed to write project name to parsed spatial file. Unable to serialize content to json file."));
```

or something similar to that